gwvet/etg.py

```diff
@@ -85,7 +85,9 @@ def get_canonical_etg_name(name):
 # register well known ETGs
 
 register_etg_parameters('omicron', **{
-    'time': 'peak',
+    # omicron hdf5 columns
+    'time': 'time',
```

Comment:
It seems like this line may not be needed because it is already defined in `_DEFAULTS`, or am I reading this wrong?

Reply:
I just added in case the default values ever change.